Code/GraphMol/Canon.cpp

```diff
@@ -240,9 +240,9 @@ void canonicalizeDoubleBond(Bond *dblBond, INT_VECT &bondVisitOrders,
 
   // now set the directionality on the other side:
   if (setFromBond1) {
-    if (dblBond->getStereo() == Bond::STEREOE) {
+    if (dblBond->getStereo() == Bond::STEREOE || dblBond->getStereo() == Bond::STEREOTRANS ) {
```

Comment:
This is repeated a lot, consider replacing with bond->isStereo() ?  This seems like a better place to encapsulate this logic.